spectrum/test_article.py

```diff
@@ -135,7 +135,7 @@ def _single_relation(from_id, to_id):
 
 @pytest.mark.journal_cms
 @pytest.mark.continuum
-def _test_adding_article_fragment(generate_article):
+def test_adding_article_fragment(generate_article):
     # TODO: publish a new article
```

Comment:
no, in the sense that now we are actually publishing a brand new article to attach a fragment to, will remove the comment